test/runnable/literal.d

```diff
@@ -174,16 +174,28 @@ void f13907_3(wchar[3] a) {}
 auto f13907_12(char[1]) { return 1; }
 auto f13907_12(char[2]) { return 2; }
 
+auto f13907_123(char[1]) { return 1; }
+auto f13907_123(char[2]) { return 2; }
+auto f13907_123(char[3]) { return 3; }
+auto f13907_123(const(char)[]) { return 0; }
+
 void test13907()
 {
     static assert(!__traits(compiles, { f13907_1("\U00010000"w); }));
     static assert(!__traits(compiles, { f13907_1("\U00010000" ); }));
     f13907_2("\U00010000"w);
     f13907_2("\U00010000");
-    static assert(!__traits(compiles, { f13907_3("\U00010000"w); }));
-    static assert(!__traits(compiles, { f13907_3("\U00010000" ); }));
+  //f13907_3("\U00010000"w);    // Re-enable implicit length extension, from issue 13999 (unlisted codegen bug)
```

Comment:
How do you produce the segfault?  What platform/args?  I'm not seeing it on win32.